derive_builder/src/options/mod.rs

```diff
@@ -454,7 +459,7 @@ impl<Mode> OptionsBuilder<Mode> where
         trace!("Parsing skip setter `{:?}`", skip);
         self.setter_enabled(!parse_lit_as_bool(skip).unwrap());
     }
-
+    
```

Comment:
If you want to review just the build function, please have a look at #70. This builds on that, so we can review and merge that one, and I'll update this to sit on top of any changes we make there.

Or did you mean the trailing whitespace?